src/epics/router.ts

```diff
@@ -130,12 +133,47 @@ const changeView: Epic = (action$, store) => action$.ofType(types.selectView)
 
 const drillDown: Epic = (action$, store) => action$.ofType(types.drillDown)
 .switchMap(action => {
-    // const state = store.getState() as Store
+    const state = store.getState() as Store
     const url = action.payload.url
     switch (action.payload.drillDownType) {
-        case 'inner':
+        case DrillDownType.external:
+            window.location.href = url
+            return Observable.empty()
```

Comment:
There is a bit of inconsistency with `return Observable.empty()` and `break` operators. We end up at `return Observable.empty()` anyway (line 179) so it probably better to just use `break` for case branches

Reply:
Fixed